data/json/scenarios.json

```diff
@@ -517,19 +517,19 @@
     "points": 3,
     "description": "Despite all the soldiers, guns and minefields, the base you were on got overrun by the dead.  Everyone was ordered to fall back to the armory, but during all the chaos you got lost and you are now stuck in the warehouse all alone.  You are not sure if anyone made it to the armory, or if you are the last man alive.",
     "start_name": "Military Base Warehouse",
-    "allowed_locs": [ "sloc_military_base_warehouse" ],
-    "professions": [ "unemployed", "soldier", "specops", "bio_soldier", "bio_sniper", "labtech", "medic", "mili_pilot" ],
+    "allowed_locs": [ "mil_base_2g" ],
+    "professions": [ "unemployed", "soldier", "specops", "bio_soldier", "bio_sniper", "labtech", "medic" ],
     "flags": [ "CHALLENGE", "LONE_START" ]
   },
   {
     "type": "scenario",
-    "ident": "presort",
-    "name": "Crazy Party",
```

Comment:
Also outside of scope(also why, I like that start 😢).

Reply:
I did not remove that

Reply:
I don't understand why that happened.